README.md

```diff
@@ -2,32 +2,63 @@
 
 This library exports a collection of exchange-rate & swap plugins for use with [`edge-core-js`](https://github.com/EdgeApp/edge-core-js).
 
-Use it like this:
+Please see [index.js](./src/index.js) for the list of plugins in this repo. These are compatible with edge-core-js v0.19.31 or later.
+
+## Installing
+
+Fist, add this library to your project:
+
+```sh
+npm i -s edge-exchange-plugins
+```
+
+### Node.js
+
+For Node.js, you should call `addEdgeCorePlugins` to register these plugins with edge-core-js:
 
 ```js
-import {
-  addEdgeCorePlugins,
-  lockEdgeCorePlugins,
-  makeEdgeContext
-} from 'edge-core-js'
-import exchangePlugins from 'edge-exchange-plugins'
-
-addEdgeCorePlugins(exchangePlugins)
+const { addEdgeCorePlugins, lockEdgeCorePlugins } = require('edge-core-js')
+const plugins = require('edge-exchange-plugins')
+
+addEdgeCorePlugins(plugins)
+
+// Once you are done adding plugins, call this:
 lockEdgeCorePlugins()
+```
 
-makeEdgeContext({
-  apiKey: '',
-  appId: '',
-  plugins: {
-    // Plugin names from edge-exchange-plugins:
-    coinbase: true,
-    shapeshift: true
-  }
+You can also add plugins individually if you want to be more picky:
+
+```js
+addEdgeCorePlugins({
+  thorchain: plugins.thorchain
 })
+```
+
+### Browser
+
+The bundle located in `dist/edge-exchange-plugins.js` will automatically register itself with edge-core-js. Just serve the entire `dist` directory along with your app, and then load the script:
 
+```html
+<script src='https://example.com/app/dist/edge-exchange-plugins.js'>
 ```
 
-Please see [index.js](./src/index.js) for the list of plugins in this repo.
+If you want to debug this project, run `yarn start` to start the a Webpack server,
```

Comment:
"the Webpack server" or "a Webpack serve"

Reply:
Also strange inconsistency to ask to run "yarn start" here, but suggest to use npm for install. Idk

Reply:
Oops, fixed.